### ISSUE TITLE
Infinite loop when calculating formulas

### ISSUE THREAD

**Issue by ses4j** [EPPlusSoftware/EPPlus#1211]
I am getting an infinite loop when running the below example.  I think it has to do with Formula parsing and quotations, but not sure.

Attached reproduction:

[infinite-loop-demo.zip](https://github.com/EPPlusSoftware/EPPlus/files/13650099/infinite-loop-demo.zip)

Run with:

`dotnet run  --infile=input.xlsx`

Here is a sample of two iterations of the infinite loop:

```
[fpl exception!] Invalid Table Formula in cell D4
OfficeOpenXml.FormulaParsing.Exceptions.InvalidFormulaException: Invalid Table Formula in cell D4
   at OfficeOpenXml.FormulaParsing.FormulaExpressions.FormulaExecutor.ExtractTableAddress(Int32 extRef, Int32 wsIx, IList`1 exps, Int32 i, FormulaTableAddress& tableAddress, ParsingContext parsingContext)
   at OfficeOpenXml.FormulaParsing.FormulaExpressions.FormulaExecutor.CompileExpressions(IList`1& tokens, ParsingContext parsingContext)
   at OfficeOpenXml.FormulaParsing.RpnFormula.SetFormula(String formula, RpnOptimizedDependencyChain depChain)
   at OfficeOpenXml.FormulaParsing.RpnFormulaExecution.GetFormula(RpnOptimizedDependencyChain depChain, ExcelWorksheet ws, Int32 row, Int32 column, Object value, RpnFormula& f)
   at OfficeOpenXml.FormulaParsing.RpnFormulaExecution.AddChainForFormula(RpnOptimizedDependencyChain depChain, RpnFormula f, ExcelCalculationOption options)
[fpl] Formula at address: Locations!D4
[fpl] Formula Tokens: SUM, Orig SOV BPP excl "Stock"  "Mobile E", ], SUM
[fpl] Formula current token : StartFunctionArguments, SUM. Position : 0
[fpl] Current Culture Setting: en-US
[fpl exception!] Invalid Table Formula in cell D4
OfficeOpenXml.FormulaParsing.Exceptions.InvalidFormulaException: Invalid Table Formula in cell D4
   at OfficeOpenXml.FormulaParsing.FormulaExpressions.FormulaExecutor.ExtractTableAddress(Int32 extRef, Int32 wsIx, IList`1 exps, Int32 i, FormulaTableAddress& tableAddress, ParsingContext parsingContext)
   at OfficeOpenXml.FormulaParsing.FormulaExpressions.FormulaExecutor.CompileExpressions(IList`1& tokens, ParsingContext parsingContext)
   at OfficeOpenXml.FormulaParsing.RpnFormula.SetFormula(String formula, RpnOptimizedDependencyChain depChain)
   at OfficeOpenXml.FormulaParsing.RpnFormulaExecution.GetFormula(RpnOptimizedDependencyChain depChain, ExcelWorksheet ws, Int32 row, Int32 column, Object value, RpnFormula& f)
   at OfficeOpenXml.FormulaParsing.RpnFormulaExecution.AddChainForFormula(RpnOptimizedDependencyChain depChain, RpnFormula f, ExcelCalculationOption options)
```

**Comment by JanKallman** [comment]
Thanks for reporting this. It seems that the tokenizer fails to handle table addresses with double quotes ("). I will supply a fix for this shortly.

**Comment by ses4j** [comment]
Your responsiveness and support are always phenomenal.  Thanks!!

**Comment by JanKallman** [comment]
Fixed in EPPlus 7.0.4
